pandas_vet/__init__.py

```diff
@@ -48,14 +50,28 @@ def check_import_name(node: ast.Import) -> List:
     return errors
 
 
-def check_inplace_false(node: ast.Import) -> List:
+def check_inplace_false(node: ast.Call) -> List:
     errors = []
     for kw in node.keywords:
         if kw.arg == "inplace" and kw.value.value is True:
             errors.append(PD002(node.lineno, node.col_offset))
     return errors
 
 
+def check_isnull_false(node: ast.Call) -> List:
```

Comment:
Bikeshed!: Maybe `check_for_isnull`? I don't think the `_false` is very descriptive.